README.md

```diff
@@ -175,12 +178,14 @@ appops set [package name] ACTIVATE_VPN allow
 | 战双帕弥什 (Punishing: Gray Raven) || 🟩 || Keyboard is supported |
 | Q-Dance | 8.0.7 | 🟥 | App crashes ||
 | QooApp | 8.3.3 | 🟩 |||
+| QQ | 8.2.11 | 🟩 |||
 | Rayman Classic | 1.0.1 | 🟩 |||
 | Reddit | | 🟩 |||
 | Relay | 10.0.378 | 🟩 |||
 | Remote Desktop (Microsoft) | 10.0.12.1148 | 🟩|||
 | Roblox | 2.499.381 | 🟨 | Graphical anomalies | GMS warnings might appear but these can be safely ignored |
 | Rootless Launcher | 3.9.1 | 🟥 | App crashes |
+| SD Maid (pro) | 5.2.2 | 🟨 | Unable to grant external storage privileges, can be skipped ||
```

Comment:
What does this exactly mean? If skipped, then the app won't be able to offer functionality regarding external storage, right? 
If so, please mention clearly.